packages/contracts-periphery/contracts/universal/op-nft/CitizenshipBadge.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import "@rari-capital/solmate/src/tokens/ERC721.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "./CitizenshipChecker.sol";
+import "./SocialContract.sol";
+
+contract CitizenshipBadge is ERC721 {
```

Comment:
None of the constructor logic sets state so this is fine to be deployed behind a delegatecall proxy, just would need to be sure the correct address is always used when interacting with the system